src/main/java/frc/team2412/robot/subsystems/ArmSubsystem.java

```diff
@@ -86,10 +86,10 @@ public static class ArmConstants {
 
 		// Trapezoid Profile
 		public static final double MAX_ARM_VELOCITY = 1;
-		public static final double MAX_ARM_ACCELERATION =
-				ARM_FORWARD_LIMIT
-						/ ARM_MOTOR_TO_SHOULDER_ENCODER_RATIO
-						/ ARM_ROTATIONS_TO_SHOULDER_ENCODER_RATIO; // arm rotations / second^2
+		public static final double MAX_ARM_ACCELERATION = 0.5;
+		// ARM_FORWARD_LIMIT
+		// 		/ ARM_MOTOR_TO_SHOULDER_ENCODER_RATIO
+		// 		/ ARM_ROTATIONS_TO_SHOULDER_ENCODER_RATIO; // arm rotations / second^2
```

Comment:
Should delete commented-out code or add an explanation as a comment why it's valuable to keep as reference